imap_processing/tests/hi/test_l1a.py

```diff
@@ -11,10 +11,12 @@ def test_sci_de_decom(create_de_data):
     """Test science direct event data"""
 
     # Process using test data
-    processed_data = hi_l1a(packet_file_path=create_de_data(HIAPID.H45_SCI_DE.value))
+    processed_data = hi_l1a(
+        packet_file_path=create_de_data(HIAPID.H45_SCI_DE.value), data_version="v001"
+    )
 
     assert processed_data[0].attrs["Logical_source"] == "imap_hi_l1a_45sensor-de"
-    assert processed_data[0].attrs["Data_version"] == "001"
+    assert processed_data[0].attrs["Data_version"] == "v001"
```

Comment:
Are we sure that we want to store "vXXX" in the `Data_version` global attribute? It seems like "v" is already implied by the attribute name.

Reply:
On second thought, I will try to convert version back to `001` and let `write_cdf` handle adding v to it to be consistent across all.

Reply:
I agree. I changed that already :)